wiki/People/The_Team/Nomination_Assessment_Team/en.md

```diff
@@ -24,7 +24,8 @@ The NAT is responsible for a variety of mapping-related tasks, with each NAT mem
 Members of the **evaluator** subgroup are primarily responsible for:
 
 - **Evaluation:** Evaluating the proficiency and activity of both current Beatmap Nominators and applicants. *See [Evaluations](/wiki/People/The_Team/Nomination_Assessment_Team/Evaluations) for more details.*
-- **Modding:** Modding at least 3 beatmaps per month. This helps evaluators keep up to date with the mapping/modding community when evaluating current and aspiring Beatmap Nominators.
+- **Nominating:** Nominating at least 2 beatmaps per month. This helps evaluators keep up to date with the mapping/modding community when evaluating current and aspiring Beatmap Nominators.
```

Comment:
```suggestion
- **Nomination:** Nominating at least 2 beatmaps per month. This helps evaluators keep up to date with the mapping/modding community when evaluating current and aspiring Beatmap Nominators.
```

Minor nitpick since we have the chance to be consistent with the word form now.